recipes/pprintpp/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pprintpp" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "06d206c43ef7c2bfdf98fe33f48d61170ab3b2b65718ec415e9bfabe887f6233" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/wolever/pprintpp/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
```

Comment:
Perhaps add a test for the command line tool and also an entry point:

https://github.com/wolever/pprintpp/blob/master/setup.py#L26